yarn.lock

```diff
@@ -5212,7 +5212,7 @@ ethereum-cryptography@^3.0.0:
     "@scure/bip32" "1.6.2"
     "@scure/bip39" "1.5.4"
 
-ethers@^5.3.1, ethers@^5.8.0:
+ethers@^5.3.1:
   version "5.8.0"
```

Comment:
@shoom3301 according to the lock file, it remains on 5.8.0. Isn't it the same?

Reply:
```
npm ls ethers
@cowprotocol/cow-sdk@6.0.0-RC.62
├─┬ @cowprotocol/app-data@3.1.0
│ └── ethers@5.8.0 deduped
├─┬ @cowprotocol/contracts@1.8.0
│ └── ethers@5.8.0 deduped
├─┬ @typechain/ethers-v5@11.1.2
│ └── ethers@5.8.0 deduped
├─┬ @weiroll/weiroll.js@0.3.0
│ └── ethers@5.8.0 deduped
└── ethers@5.8.0

```

@alfetopito it gets the version from many other packages, the most imortant is that it's defined in `peerDependencies` and not in `dependencies`.
I've tested this version of SDK in https://github.com/cowprotocol/cowswap/pull/5831 and it looks good.